Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_47.md

```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Cortex ASM - Azure Enrichment
+
+Fixed an issue in the playbook where Azure Hierarchy Hierarchy task's default path was not handled correctly.
```

Comment:
```suggestion
Fixed an issue in the playbook where the Azure Hierarchy Hierarchy task's default path was not handled correctly.
```

Reply:
Thank you for the suggestion. Updated release notes.